openvdb_houdini/houdini/ParmFactory.h

```diff
@@ -546,6 +546,7 @@ class OPENVDB_HOUDINI_API OpPolicy
 
 
 /// @brief Helper class to manage input locking.
+/// @deprecated Verbification renders this redundant (recommend using OP_AutoLockInputs directly)
```

Comment:
I thought we agreed that `ScopedInputLock` would not be deprecated.  Verbification might render it redundant, but not all of our SOPs are verbified.  I see no reason to stop using it in those SOPs.

Reply:
Yes, my mistake. Tried again with the update - removed the deprecated comments and reverted back to using ScopedInputLock.